CHANGELOG.md

```diff
@@ -1,8 +1,299 @@
+## ClickHouse release v20.4
+
+### ClickHouse release v20.4.2.9, 2020-05-12
+
+#### Backward Incompatible Change
```

Comment:
Please add a note that system tables (like query_log, metric_log, trace_log) now using compact parts format,
and in case of downgrade to versions less than 20.3,
the user has to manually delete the data for these tables.